esp-hal-embassy/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![no_std]
+#![cfg_attr(
+    any(feature = "esp32", feature = "esp32s2", feature = "esp32s3"),
+    feature(asm_experimental_arch)
+)]
+#![doc(html_logo_url = "https://avatars.githubusercontent.com/u/46717278")]
+
+use core::cell::Cell;
+
+#[cfg(any(
+    feature = "executor-thread",
+    feature = "time-systimer",
+    feature = "time-timg0"
+))]
+use esp_hal_common::{clock::Clocks, peripherals, trapframe};
+pub use procmacros::main;
+
+// MUST be the first module
+mod fmt;
+
+pub mod executor;
+mod time_driver;
+
+/// TODO: document me
+#[cfg(any(feature = "time-systimer", feature = "time-timg0"))]
```

Comment:
Is there a valid configuration where neither of these are active?

Reply:
Nope, just forgot to add the build script and that was temporarily added to silence some warnings. Thanks for the reminder, will get that fixed tomorrow.